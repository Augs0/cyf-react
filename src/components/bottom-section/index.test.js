import React from 'react';
import { shallow } from 'enzyme';
import BottomSection from './';

describe('BottomSection', () => {
  it('renders correctly', () => {
    expect(
      shallow(
        <BottomSection link="/apply/volunteer" linkText="Become a Volunteer" />,
      ),
    ).toMatchSnapshot();
  });
});
