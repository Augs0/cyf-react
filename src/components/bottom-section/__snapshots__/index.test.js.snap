// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomSection renders correctly 1`] = `
<div
  className="section-bottom-link"
>
  <Link
    className="big-link-3 btn"
    rel="noopener noreferrer"
    replace={false}
    target="_blank"
    to="/apply/volunteer"
  >
    Become a Volunteer
  </Link>
</div>
`;
